GVRf/Framework/framework/build.gradle

```diff
@@ -73,7 +76,7 @@ android {
     }
 
     sourceSets.main {
-        java.srcDirs = ['src/main/java', 'src/main/backends/oculus']
+        java.srcDirs = ['src/main/java', '../' + rootProject.property("GVRF_BACKEND")+'/src/main/java']

```

Comment:
Does this enforce backend 2 to keep the same file names to support all apps that work with backend 1?